contracts/oracle/Cargo.toml

```diff
@@ -19,3 +19,5 @@ sbt = { path = "../sbt" }
 
 [dev-dependencies]
 rand = "^0.7"
+base64 = "0.13"
```

Comment:
we don't need this package. See how base64 is done in other functions. near-sdk already re-exports base64 functions.